server/statistics/kind_test.go

```diff
@@ -45,24 +44,24 @@ func (s *testRegionInfoSuite) TestGetLoads(c *C) {
 		core.SetWrittenKeys(4),
 		core.SetQueryStats(queryStats))
 	loads := regionA.GetLoads()
-	c.Assert(loads, HasLen, int(RegionStatCount))
-	c.Assert(float64(regionA.GetBytesRead()), Equals, loads[RegionReadBytes])
-	c.Assert(float64(regionA.GetKeysRead()), Equals, loads[RegionReadKeys])
-	c.Assert(float64(regionA.GetReadQueryNum()), Equals, loads[RegionReadQuery])
+	re.Len(loads, int(RegionStatCount))
+	re.Equal(loads[RegionReadBytes], float64(regionA.GetBytesRead()))
```

Comment:
```suggestion
	re.Equal(float64(regionA.GetBytesRead()),loads[RegionReadBytes],)
```
 as the function `Equal` defines: ref [https://github.com/stretchr/testify/blob/master/require/require_forward.go#L121-L128](https://github.com/stretchr/testify/blob/master/require/require_forward.go#L121-L128)
``` 
Equal(expected interface{}, actual interface{}, msgAndArgs ...interface{}) 
```